### PR TITLE
Write-the-docs-2020-#5664-block#1

### PR DESCRIPTION
**Pull request by AstroChoco**
#5664

### CODE REVIEW

**Comment by PRMerger7** [comment]
@AstroChoco : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/analyzer-rule-sets.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ The process for setting the active rule set is a little different depending on w
 
 #### .NET Core
 
-To make a rule set the active rule set for analysis in .NET Core or .NET Standard projects, manually add the **CodeAnalysisRuleSet** property to your project file. For example, the following code snippet sets `HelloWorld.ruleset` as the active rule set.
+To make a rule, set the active rule set for analysis in .NET Core or .NET Standard projects, manually add the **CodeAnalysisRuleSet** property to your project file. For example, the following code snippet sets `HelloWorld.ruleset` as the active rule set.
```
Comment (mavasani) [inline docs/code-quality/analyzer-rule-sets.md:68]: Please revert, and instead use “ruleset”

File: docs/code-quality/analyzer-rule-sets.md [hunk 2]
```diff
@@ -76,21 +76,21 @@ To make a rule set the active rule set for analysis in .NET Core or .NET Standar
 
 #### .NET Framework
 
-To make a rule set the active rule set for analysis in .NET Framework projects:
+To make a rule, set the active rule set for analysis in .NET Framework projects:
```
Comment (mavasani) [inline docs/code-quality/analyzer-rule-sets.md:79]: Revert

**Comment by ktoliver** [comment]
@Mikejo5000 @ghogen - Multiple revisions in the PR don't align with the Microsoft Style Guide. Should the PR review team move the commit to the private repo and make revisions? With a commit move, the original contributor remains associated with the update, but we get the advantage of using Acrolinx and automated validation.

**Comment by Mikejo5000** [comment]
In discussion, but for VS right-click is likely the preferred first option vs. select-and-hold, which is more difficult. Some of these will likely need edits.

**Comment by AstroChoco** [comment]
> In discussion, but for VS right-click is likely the preferred first option vs. select-and-hold, which is more difficult. Some of these will likely need edits.
I think the action verbs may depend on how aid tools interact with the described service and product. I'd be happy to update my PR once you have come to a decision. :)

File: docs/code-quality/analyzer-rule-sets.md (hunk 1 above)
Comment (AstroChoco) [reply docs/code-quality/analyzer-rule-sets.md:68]: Will fix it. Thank you for reviewing!

**Comment by PRMerger19** [comment]
@AstroChoco : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/analyzer-rule-sets.md (hunk 1 above)
Comment (AstroChoco) [reply docs/code-quality/analyzer-rule-sets.md]: Hi, again, @mavasani, I noticed there are a few UI elements with "ruleset" named differently:
- **Run this ruleset**
- **CodeAnalysisRuleSet**

Which spelling is correct: ruleset or RuleSet?

**Comment by AstroChoco** [comment]
> @Mikejo5000 @ghogen - Multiple revisions in the PR don't align with the Microsoft Style Guide. Should the PR review team move the commit to the private repo and make revisions? With a commit move, the original contributor remains associated with the update, but we get the advantage of using Acrolinx and automated validation.

Hi @ktoliver , would you point out which revisions don't alight with MS style guide? I'd like to fix them.

**Comment by Mikejo5000** [comment]
@AstroChoco We discussed this yesterday. For VS, touch screen usage is believed to be quite low, so instead of "select and hold (or right-click)" we would want to say "right-click (or select and hold)" instead. There is a question over "select" vs. "choose" for VS docs, but I believe we will fix these later, if needed. @ktoliver for input if there are other issues that don't match the style guide. @ghogen as well.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md [hunk 3]
```diff
@@ -61,19 +61,19 @@ If you enable remote debugging for a cloud service, it doesn't exhibit degraded
 
 ### To attach the debugger to a cloud service in Azure
 
-1. In Server Explorer, expand the node for your cloud service.
+1. In **Server Explorer**, expand the node for your cloud service.
```
Comment (ktoliver) [inline docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:64]: This should remain plain text. Applications and their components (windows, explorers) should be plain text unless you are describing an action like "select **Server Explorer**, and then select . . ."

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md [hunk 4]
```diff
@@ -61,19 +61,19 @@ If you enable remote debugging for a cloud service, it doesn't exhibit degraded
...
 
     ![Attach Debugger](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC746719.gif)
 
-3. After the debugger attaches to an instance, debug as usual. The debugger automatically attaches to the appropriate host process for your role. Depending on what the role is, the debugger attaches to w3wp.exe, WaWorkerHost.exe, or WaIISHost.exe. To verify the process to which the debugger is attached, expand the instance node in Server Explorer. See [Azure Role Architecture](https://blogs.msdn.microsoft.com/kwill/2011/05/05/windows-azure-role-architecture/) for more information about Azure processes.
+3. After the debugger attaches to an instance, debug as usual. The debugger automatically attaches to the appropriate host process for your role. Depending on what the role is, the debugger attaches to w3wp.exe, WaWorkerHost.exe, or WaIISHost.exe. To verify the process to which the debugger is attached, expand the instance node in **Server Explorer**. See [Azure Role Architecture](https://blogs.msdn.microsoft.com/kwill/2011/05/05/windows-azure-role-architecture/) for more information about Azure processes.
```
Comment (ktoliver) [inline docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:72]: Use plain text for Server Explorer in this context.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md [hunk 5]
```diff
@@ -61,19 +61,19 @@ If you enable remote debugging for a cloud service, it doesn't exhibit degraded
...
 
     ![Select code type dialog box](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC718346.png)
 
-4. To identify the processes to which the debugger is attached, open the Processes dialog box by, on the menu bar, choosing Debug, Windows, Processes. (Keyboard: Ctrl+Alt+Z) To detach a specific process, open its shortcut menu, and then select **Detach Process**. Or, locate the instance node in Server Explorer, find the process, open its shortcut menu, and then select **Detach Process**.
+4. To identify the processes to which the debugger is attached, go to the **Processes** dialog box by, on the menu bar, selecting **Debug**, **Windows**, **Processes**. (Keyboard: Ctrl+Alt+Z) To detach a specific process, open its shortcut menu, and then select **Detach Process**. Or, locate the instance node in **Server Explorer**, find the process, open its shortcut menu, and then select **Detach Process**.
```
Comment (ktoliver) [inline docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:76]: I would do this:

_To identify the processes that are attached to the debugger, on the menu bar, select **Debug** > **Windows** > **Processes**, and then open the **Processes** dialog box. (Keyboard: Ctrl+Alt+Z) To detach a specific process, open its shortcut menu, and then select **Detach Process**. Or, locate the instance node in Server Explorer, find the process, open its shortcut menu, and then select **Detach Process**._

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md [hunk 6]
```diff
@@ -102,27 +102,27 @@ You can debug programs that run on Azure virtual machines by using Server Explor
 
 ### To debug an Azure virtual machine
 
-1. In Server Explorer, expand the Virtual Machines node and select the node of the virtual machine that you want to debug.
+1. In **Server Explorer**, expand the Virtual Machines node and select the node of the virtual machine that you want to debug.
```
Comment (ktoliver) [inline docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:105]: Keep original.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md [hunk 7]
```diff
@@ -102,27 +102,27 @@ You can debug programs that run on Azure virtual machines by using Server Explor
...
 
-2. Open the context menu and select **Enable Debugging**. When asked if you're sure if you want to enable debugging on the virtual machine, select **Yes**.
+2. Go to the context menu and select **Enable Debugging**. When asked if you're sure if you want to enable debugging on the virtual machine, select **Yes**.
```
Comment (ktoliver) [inline docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:107]: I would keep the original.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md [hunk 8]
```diff
@@ -102,27 +102,27 @@ You can debug programs that run on Azure virtual machines by using Server Explor
...
 
     Azure installs the remote debugging extension on the virtual machine to enable debugging.
 
     ![Virtual machine enable debugging command](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC746720.png)
 
     ![Azure activity log](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC746721.png)
 
-3. After the remote debugging extension finishes installing, open the virtual machine's context menu and select **Attach Debugger...**
+3. After the remote debugging extension finishes installing, go to the virtual machine's context menu and select **Attach Debugger...**
```
Comment (ktoliver) [inline docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:115]: I would keep the original.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md [hunk 9]
```diff
@@ -143,17 +143,17 @@ Visual Studio ASP.NET projects offer an option to create a handy virtual machine
...
 
     The name you enter for DNS name will be the name of the virtual machine.
 
     ![Create virtual machine on Azure dialog box](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC746724.png)
 
-    Azure creates the virtual machine and then provisions and configures the endpoints, such as Remote Desktop and Web Deploy
+    Azure creates the virtual machine and then provisions and configures the endpoints, such as Remote Desktop and Web Deploy.
 
-4. After the virtual machine is fully configured, select the virtual machine’s node in Server Explorer.
+4. After the virtual machine is fully configured, select the virtual machine’s node in **Server Explorer**.
```
Comment (ktoliver) [inline docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:154]: Keep original.

File: docs/azure/vs-azure-tools-resources-managing-with-cloud-explorer.md [hunk 10]
```diff
@@ -38,11 +38,11 @@ To view the resources associated with an Azure account, you must first add the a
 
 1. Log in to the Azure account whose resources you want to browse.
 
-1. Once logged in to an Azure account, the subscriptions associated with that account display. Select the check boxes for the account subscriptions you want to browse and then select **Apply**.
+1. Once logged in to an Azure account, the subscriptions associated with that account display. Choose the check boxes for the account subscriptions you want to browse and then select **Apply**.
```
Comment (ktoliver) [inline docs/azure/vs-azure-tools-resources-managing-with-cloud-explorer.md:41]: Microsoft style is to use "select" for a check box.

File: docs/code-quality/analyzer-rule-sets.md [hunk 11]
```diff
@@ -1,10 +1,10 @@
 ---
-title: FxCop analyzer rule sets and editorconfig files
+title: FxCop analyzer rulesets and editorconfig files
```
Comment (ktoliver) [inline docs/code-quality/analyzer-rule-sets.md:2]: Neither "ruleset" nor "rule set" are included in Microsoft style guides, so I'd leave as two words, depending on the preference of the content team.

**Comment by ktoliver** [comment]
> @AstroChoco We discussed this yesterday. For VS, touch screen usage is believed to be quite low, so instead of "select and hold (or right-click)" we would want to say "right-click (or select and hold)" instead. There is a question over "select" vs. "choose" for VS docs, but I believe we will fix these later, if needed. @ktoliver Kristine Toliver (Aquent LLC) Vendor for input if there are other issues that don't match the style guide. @ghogen Gordon Hogenson FTE as well.

@Mikejo5000 I added comments inline to make it easier to describe what I'm referring to. Other items I notice are _choose_ versus _select_.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md (hunk 5 above)
Comment (AstroChoco) [reply docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:76]: Hi, there isn't a contextual screenshot or an environment for me to be sure about the relations among the UI elements. I assume you have checked the actual UI. So I'll make the update as per your suggestion.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md (hunk 6 above)
Comment (AstroChoco) [reply docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:105]: Thank you! :) I'll keep the title-style capitalization only.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md (hunk 8 above)
Comment (AstroChoco) [reply docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md:115]: The guideline about using "go to" to open menus is here: https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui.
Does it apply in this context?

File: docs/azure/vs-azure-tools-resources-managing-with-cloud-explorer.md (hunk 10 above)
Comment (AstroChoco) [reply docs/azure/vs-azure-tools-resources-managing-with-cloud-explorer.md:41]: Thank you! :) I mistook "check boxes" here for UI options. I'll fix it.

File: docs/code-quality/analyzer-rule-sets.md (hunk 11 above)
Comment (AstroChoco) [reply docs/code-quality/analyzer-rule-sets.md:2]: Hi, @mavasani, could you double check?

Comment (Mikejo5000) [reply docs/code-quality/analyzer-rule-sets.md:2]: Yes, "rule set" is the current spelling in the docs. Please stick with that. Thanks!

File: docs/code-quality/analyzer-rule-sets.md (hunk 1 above)
Comment (Mikejo5000) [reply docs/code-quality/analyzer-rule-sets.md]: "rule set" is the current spelling in the docs. Thanks.

File: docs/code-quality/analyzer-rule-sets.md [hunk 12]
```diff
@@ -1,10 +1,10 @@
...
 ms.date: 10/08/2019
 ms.topic: conceptual
 helpviewer_keywords:
-- analyzer packages, rule sets
-- rule sets for analyzers
+- analyzer packages, rulesets
```
Comment (Mikejo5000) [inline docs/code-quality/analyzer-rule-sets.md:6]:
```suggestion
- analyzer packages, rule sets
```

File: docs/code-quality/analyzer-rule-sets.md [hunk 13]
```diff
@@ -1,10 +1,10 @@
...
+- rulesets for analyzers
```
Comment (Mikejo5000) [inline docs/code-quality/analyzer-rule-sets.md:7]:
```suggestion
- rule sets for analyzers
```

File: docs/code-quality/analyzer-rule-sets.md (hunk 11 above)
Comment (Mikejo5000) [inline docs/code-quality/analyzer-rule-sets.md:2]:
```suggestion
title: FxCop analyzer rule sets and editorconfig files
```

**Comment by PRMerger9** [comment]
@AstroChoco : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md (hunk 3 above)
Comment (AstroChoco) [reply docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md]: hi, @ktoliver , sorry to check with you again. I reverted all the bold face I added on "server explorer". However, I noticed that some contexts like this one specifically are in interactions with the UI. Even they are not selected but it's the interface of Server Explorer where users interact. Please see many original occurrences in block#1 files. For example, **Cloud Explorer** in docs/azure/vs-azure-tools-resources-managing-with-cloud-explorer.md.
I'd think such alignment of using bold face for UI elements need your discussion with your team before I make further changes.

**Comment by PRMerger13** [comment]
@ghogen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ghogen** [comment]
We're going to merge these WriteTheDocs PRs now and go through a pass on them in the private repo to address any issues.  This one looks quite good, thanks for all the work on it @AstroChoco @ktoliver @Mikejo5000

**Pull request merged**